localedir.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/scripts/picard.in b/scripts/picard.in
+index 7580da67..18b7f02a 100644
+--- a/scripts/picard.in
++++ b/scripts/picard.in
+@@ -1,3 +1,4 @@
+ #!/usr/bin/env python3
+-from picard.tagger import main; main('%(localedir)s', %(autoupdate)s)
+-
++import os
++localdir = os.path.join(os.environ.get('SNAP', ''), '%(localedir)s')
```

Comment:
Thanks for this! Just one thing: shouldn't it be `localedir = ...`?
```suggestion
+localedir = os.path.join(os.environ.get('SNAP', ''), '%(localedir)s')
```

Reply:
Thanks, good catch. Fixed